python/tests/orderby_tests/orderby_int_varchar.py

```diff
@@ -0,0 +1,120 @@
+from tests.aggregate_tests.aggtst_base import TstView
+
+
+class orderby_asc_int_asc_varchar(TstView):
+    def __init__(self):
+        # Validated on SQLite
+        self.data = []
+        self.sql = """CREATE MATERIALIZED VIEW orderby_asc_int_asc_varchar AS
+                       SELECT c1, c2
+                       FROM orderby_tbl_int_varchar
+                       ORDER BY c1 ASC, c2 ASC
+                       LIMIT 3"""
+        # Returns the following:
```

Comment:
these comments are not necessary
the whole point of running automatically on sqlite is that you don't have to write this anymore.